lib/server.rb

```diff
@@ -87,6 +87,18 @@ def pluralize_scope(object, singular, plural)
       object.length.to_s + " " + Machinery.pluralize(object.length, singular, plural)
     end
 
+    def key_value_to_cell(key, value)
+      cell = "<td>"
+      if key == "url"
+        cell += "<a href='#{value}'>#{value}</a>"
+      elsif value.is_a?(Machinery::Array)
+        cell += value.join(", ")
+      else
+        cell  += value.to_s
```

Comment:
Unnecessary spacing detected.<br>Operator `+=` should be surrounded with a single space.